app/queue_consumers/tagging_updater.rb

```diff
@@ -0,0 +1,44 @@
+# TaggingUpdater
+#
+# To allow us to migrate to the new tagging infrastructure, we need to have
+# taggings be synced between panopticon and publishing-api. This class makes
+# sure that if an app sends `links` to the publishing-api, panopticon/contentapi
+# is also updated.
+class TaggingUpdater
+  def process(message)
+    content_item = message.payload
+
+    return message.ack unless content_item['publishing_app'].in?(Settings.apps_with_migrated_tagging)
```

Comment:
This code works correctly, I believe, but makes it look like what matters is that the result of calling "message.ack" is returned.  Actually, what matters is that message.ack is called, and then the method returns.

I think this intent would be more clearly expressed by returning nil from this method in all cases, and changing the unless clause to a multi-line conditional: call "message.ack" on one line, then return on the next.
```
unless content_item['publishing_app'].in?(Settings.apps_with_migrated_tagging)
  message.ack
  return
end
```